service/models/operations/ensemble_simulate.py

```diff
@@ -48,14 +57,24 @@ def gen_pyciemss_args(self, job_id):
             extra_options.pop("inferred_parameters"), job_id
         )
 
+        solver_options = {}
+        step_size = extra_options.pop(
+            "solver_step_size"
+        )  # Need to pop this out of extra.
```

Comment:
Let's remove this comment "Need to pop this out of extra."?? Because you are popping a bunch of other entries and I don't know why this one is so special that it needs to be called out? Or am I missing something here?

Reply:
I did not make this comment clear I will remove it as it adds more confusion than helps.

Here is what I was trying to portray:

On line 79 we are provided whatever else is leftover in `extra_options` 
We do not want to provide `step_size` in this section as it is only relevant when we pick `euler`.
I thought that this may cause confusion as to why we are popping it out of extras all of the time (before if), but only using it some of the time (within if)